kolibri/core/sqlite/pragmas.py

```diff
@@ -1,3 +1,3 @@
-CONNECTION_PRAGMAS = "PRAGMA wal_autocheckpoint=500;"
+CONNECTION_PRAGMAS = "PRAGMA wal_autocheckpoint=500; PRAGMA legacy_alter_table = ON;"
```

Comment:
I'd like to know if this has been tested in previous versions of Debian and Ubuntu. Looking at the [sqlite3 changelog](https://metadata.ftp-master.debian.org/changelogs//main/s/sqlite3/sqlite3_3.27.2-3_changelog)  I see sqlite 3.26 entered into Debian unstable on Dec 2018, what means it entered into stable with Debian Buster (Jul 2019) . If the pragma was not available in previous versions we need to know if it will trigger an application error.

Reply:
thank you! yes, this makes sense to me. sorry I forgot to test if this change would break the old sqlite versions. I will test it and report here soon!

Reply:
@jredrejo I tested on Ubuntu 16.04 with sqlite 3.11.0 - the test passed without an error, and kolibri starts successfully. Do you have any suggestions if there is anything else I should test? Thank you!

Reply:
Nope, I don't have any other concern. Thanks!